winmake.ps1

```diff
@@ -44,6 +44,12 @@ function Make-Clean{
     }
 }
 
+function Local-Unit {
+    Build-Ginkgo
+    $skippackages="pkg\machine\apple,pkg\machine\applehv,pkg\machine\e2e,pkg\machine\libkrun,pkg\machine\provider,pkg\machine\proxyenv,pkg\machine\qemu"
+    Run-Command "./test/tools/build/ginkgo.exe -vv -r --tags `"$remotetags`" -timeout=90m --trace --no-color --skip-package `"$skippackages`" pkg\machine"
```

Comment:
Shouldn't we run all unit tests on windows not just pkg/machine? unit tests could be added anywhere.
point in case https://github.com/containers/podman/pull/25323


If these issue is that these do not pass then we need to do the hard work and have !windows build tags for them.

Reply:
Thank you for the review. I have changed that to run the tests for every package.

Packages with compilation errors are skipped and failing tests (3 files) have now either `!windows` tag or `if runtime.GOOS == "windows" t.Skip()`.